src/Microsoft.AspNet.Mvc/MvcServiceCollectionExtensions.cs

```diff
@@ -20,6 +17,28 @@ public static IServiceCollection AddMvc(this IServiceCollection services, IConfi
             return services;
         }
 
+        /// <summary>
+        /// Configures a set of MvcOptions for the application.
+        /// </summary>
+        /// <param name="services">The services available in the application.</param>
+        /// <param name="setupAction">The MvcOptions which need to be configured.</param>
+        public static void ConfigureMvcOptions(
+            this IServiceCollection services, [NotNull]Action<MvcOptions> setupAction)
+        {
+            services.Configure(setupAction);
+        }
+
+        /// <summary>
+        /// Configures a set of RouteOptions for the application.
+        /// </summary>
+        /// <param name="services">The services available in the application.</param>
+        /// <param name="setupAction">The RouteOptions which need to be configured.</param>
+        public static void ConfigureRouteOptions(
```

Comment:
Why is this defined in MVC? This really seems like it should be defined in routing.